modules/juce_audio_devices/native/juce_android_Audio.cpp

```diff
@@ -22,6 +22,22 @@
   ==============================================================================
 */
 
+//==============================================================================
+#define JNI_CLASS_MEMBERS(METHOD, STATICMETHOD, FIELD, STATICFIELD) \
+ STATICMETHOD (nanoTime, "nanoTime", "()J") \
+
+DECLARE_JNI_CLASS (System, "java/lang/System");
+#undef JNI_CLASS_MEMBERS
+
+//==============================================================================
+#define JNI_CLASS_MEMBERS(METHOD, STATICMETHOD, FIELD, STATICFIELD) \
+ METHOD (constructor,   "<init>",        "()V") \
+ FIELD  (framePosition, "framePosition", "J") \
+ FIELD  (nanoTime,      "nanoTime",      "J") \

```

Comment:
Do the two lines above work without the brackets?

Reply:
Yes, that's the format for fields, specifically.